### PR TITLE
API-8648: openapi spec package

### PR DESCRIPTION
**Pull request by jabando-va**

### CODE REVIEW

File: patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/ObservationApi.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package gov.va.api.health.patientgenerateddata.observation;
+package gov.va.api.health.patientgenerateddata.api;
```
Comment (jhulbert-va) [inline patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/ObservationApi.java:1]: Can we make this `R4ObservationApi`?

Comment (apeirats-va) [reply patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/ObservationApi.java:1]: PGD only deals with R4 so we figured it was redundant

File: patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/OpenApi.java [hunk 2]
```diff
@@ -1,8 +1,5 @@
-package gov.va.api.health.patientgenerateddata;
+package gov.va.api.health.patientgenerateddata.api;
```
Comment (jhulbert-va) [inline patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/OpenApi.java:1]: `R4PatientGeneratedDataService`?

File: patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/QuestionnaireApi.java [hunk 3]
```diff
@@ -1,4 +1,4 @@
-package gov.va.api.health.patientgenerateddata.questionnaire;
+package gov.va.api.health.patientgenerateddata.api;
```
Comment (jhulbert-va) [inline patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/QuestionnaireApi.java:1]: R4

File: patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/QuestionnaireResponseApi.java [hunk 4]
```diff
@@ -1,4 +1,4 @@
-package gov.va.api.health.patientgenerateddata.questionnaireresponse;
+package gov.va.api.health.patientgenerateddata.api;
```
Comment (jhulbert-va) [inline patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/QuestionnaireResponseApi.java:1]: R4

File: patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/ObservationApi.java (hunk 1 above)
Comment (jhulbert-va) [reply patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/ObservationApi.java:1]: I think we should refactor for two reasons:
1. Uniformity. Both DQ and VFQ define the fhir versions in the class name (VFQ is also exclusively R4).
2. If there was a need for a different version, no refactor would be required here.

Comment (apeirats-va) [reply patient-generated-data/src/main/java/gov/va/api/health/patientgenerateddata/api/ObservationApi.java:1]: Ok. I'm happy to keep consistency with DQ and VFQ.

**Review by jhulbert-va — APPROVED** [review]
Looks good to me.

**Review by jblefkove-va — APPROVED** [review]

**Review by apeirats-va — APPROVED** [review]

**Pull request merged**